src/main/java/com/conveyal/r5/analyst/cluster/PathWriter.java

```diff
@@ -33,22 +32,26 @@
 
     OutputStream outputStream;
 
-    public PathWriter (AnalysisTask task, TransportNetwork network, List<Path> pathList) {
+    public PathWriter (AnalysisTask task, TransportNetwork network, List<Path> pathList, int destinations, int iterations) {
         this.task = task;
         this.network = network;
         this.pathList = pathList;
         try {
             outputStream = MultipointDataStore.getOutputStream(task, task.taskId + "_paths.dat", "application/octet-stream");
             outputStream.write("PATHGRID".getBytes());
+            // In the header store the number of destinations stored and iterations used
```

Comment:
I would mention that the iterations are "inside" the destinations, e.g. `the number of destinations and the number of iterations at each destination`.